app/components/provider_interface/interview_and_course_summary_component.html.erb

```diff
@@ -5,10 +5,16 @@
         <div class="app-summary-card__actions">
           <ul class="app-summary-card__actions-list">
             <li class="app-summary-card__actions-list-item">
-              <%= govuk_link_to 'Change details', edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link--no-visited-state' %>
+              <%= link_to edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link govuk-link--no-visited-state' do %>
+                <span class="govuk-visually-hidden"> <%= interview.date_and_time.to_s(:govuk_date_and_time) %> </span>
+                Change details
+              <% end %>
             </li>
             <li class="app-summary-card__actions-list-item">
-              <%= govuk_link_to 'Cancel interview', cancel_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link--no-visited-state' %>
+              <%= link_to cancel_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link govuk-link--no-visited-state' do %>
+                <span class="govuk-visually-hidden"> <%= interview.date_and_time.to_s(:govuk_date_and_time) %> </span>
+                Cancel interview
```

Comment:
In the prototype the hidden span is placed after the visible link text. We use this pattern when making all links unique.
<img width="520" alt="Screenshot 2021-02-23 at 12 34 10" src="https://user-images.githubusercontent.com/38078064/108844263-c8106900-75d3-11eb-8234-fc4a3891c469.png">